x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/date_histogram.tsx

```diff
@@ -334,3 +340,77 @@ function restrictedInterval(aggregationRestrictions?: Partial<IndexPatternAggRes
     aggregationRestrictions.date_histogram.fixed_interval
   );
 }
+
+const AutoDateHistogramPopover = ({ data }: { data: DataPublicPluginStart }) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const infiniteBound = i18n.translate('xpack.lens.indexPattern.dateHistogram.moreThanYear', {
+    defaultMessage: 'More than a year',
+  });
+  const upToLabel = i18n.translate('xpack.lens.indexPattern.dateHistogram.upTo', {
+    defaultMessage: 'Up to',
+  });
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+          {i18n.translate('xpack.lens.indexPattern.dateHistogram.autoHelpText', {
+            defaultMessage: 'How does it work?',
+          })}
+        </HelpPopoverButton>
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.lens.indexPattern.dateHistogram.titleHelp', {
+        defaultMessage: 'How does the auto date histogram work?',
+      })}
+    >
+      <p>
+        {i18n.translate('xpack.lens.indexPattern.dateHistogram.autoBasicExplanation', {
+          defaultMessage: 'Splits a date field into buckets by interval.',
+        })}
+      </p>
+
+      <p>
+        <FormattedMessage
+          id="xpack.lens.indexPattern.dateHistogram.autoLongerExplanation"
+          defaultMessage="The Lens editor chooses an automatic interval for you by dividing the selected time range by the 
+                  {targetBarSetting} Kibana advanced setting. The calculation tries to present “nice” time interval buckets. The maximum 
```

Comment:
```suggestion
                  {targetBarSetting} advanced setting. **Lens** displays the time interval buckets that match your data. The maximum 
```

Reply:
I see the problem here, same as with the `"nice"` definition below, but the new version

>> **Lens** displays the time interval buckets that match your data

I am not sure it is correct or add any particular other value in this context. The table below should already clarify in terms of time magnitude what "nice" means in here. I would probably prefer to have a documentation page with a full explanation of the logic behind the algorithm to point at, rather than make a short and approximate explanation here.